src/protagonist/DLCS.Repository/Messaging/EngineClient.cs

```diff
@@ -133,26 +137,20 @@ public class EngineClient : IEngineClient
 
         return overallSent;
     }
-
-    private async Task<string> GetJsonString(IngestAssetRequest ingestAssetRequest, bool derivativesOnly)
+    
+    private async Task<string> GetJsonString(IngestAssetRequest ingestAssetRequest)

```

Comment:
was struggling to see what you meant, but think I get it now -yeah I can make it so that `GetJsonString` is responsible for creating the `IngestAssetRequest` - additionally, I don't think this method needs to be `async` anymore - will get that updated as well